resources/public/api/conf/2.0/schemas/retrieveBalanceAndTransactions/financialDetails.json

```diff
@@ -0,0 +1,107 @@
+            "type":"string",
+            "example":"2020-21"
+          },
+          "chargeDetail":{
+            "type":"object",
+            "properties":{
+              "documentId":{
+                "type":"string",
+                "description":"A unique ID is used to reference the document of which this is a subitem.\nMust conform to the regular expression: ^[0-9a-zA-Z]{1,30}$",
+                "examples":"XAIS12345678910"
+              },
+              "documentType":{
+                "type":"string",
+                "description":"A unique ID is used to reference the document type of which this is a subitem.",
+                "examples":"1234"
+              },
+              "documentTypeDescription":{
+                "type":"string",
+                "description":"The Description of the main charge type",
+                "examples":"SA Balancing Charge"
+              },
+              "chargeType":{
+                "type":"string",
+                "description":"A unique ID is used to identify the type of sub-item.",
+                "examples":"3175"
+              },
+              "chargeTypeDescription":{
+                "type":"string",
+                "description":"Name of charge type.\nMust conform to the regular expression:\n^[0-9A-Za-z]{1,30}$",
+                "examples":"Income Tax Estimate - ITSA"
+              }
+            },
+            "required":[
+              "documentId"
+            ]
+          },
+          "taxPeriodFrom":{
+            "type":"string",
+            "description":"The 'from' date of this tax period.\nDate in the format: YYYY-MM-DD",
+            "examples":"2020-04-06"
+          },
+          "taxPeriodTo":{
+            "type":"string",
+            "description":"The 'to' date of this tax period.\nDate in the format: YYYY-MM-DD",
+            "examples":"2021-04-05"
+          },
+          "contractAccount":{
+            "type":"string",
+            "description":"A contract account is the accounting record for a particular charge type.",
+            "examples":"X"
+          },
+          "documentNumber":{
+            "type":"string",
+            "description":"Same as Document Number/ID unless on output there is no postings (zero amounts) then the Form Bundle is returned \n\nMust conform to the regular expression: ^[0-9A-Za-z]{1,12}$",
+            "examples":"1040000872"
+          },
+          "documentNumberItem":{
+            "type":"string",
+            "description":"An identifier for this part of the overall transaction is used to link an allocation and charge.",
+            "examples":"XM00"
+          },
+          "chargeReference":{
+            "type":"string",
+            "description":"A unique reference number is associated with the charge.",
+            "examples":"XM002610011594"
+          },
+          "originalAmount":{
+            "type":"integer",
```

Comment:
I think everywhere it says "integer" should be replaced with "number" .

5009.99 is not an integer